CHANGELOG.md

```diff
@@ -78,6 +78,8 @@ defaults:
 * [BUGFIX] Moved empty root span substitution from `querier` to `query-frontend`. [#2671](https://github.com/grafana/tempo/issues/2671) (@galalen)
 * [BUGFIX] Correctly propagate ingester errors on the query path [#2935](https://github.com/grafana/tempo/issues/2935) (@joe-elliott)
 * [BUGFIX] Fix cases where empty filter {} wouldn't return expected results [#2498](https://github.com/grafana/tempo/issues/2498) (@mdisibio)
+* [BUGFIX] Reorder S3 credential chain and upgrade minio-go [#3006](https://github.com/grafana/tempo/pull/3006) (@ekristen, @mapno)
+    Removes config option `native_aws_auth_enabled`. If you were using this option, remove it from your config.
```

Comment:
is this a breaking change? if yes, can we please mark it as breaking change here?

Reply:
Removing `native_aws_auth_enabled` if left in the config will cause a break due to how the config is parsed. I would suggest that perhaps we leave the config option in and write a log message if it's set that it's deprecated and remove it in the next major release.

Reply:
Leaving it in place and saying it's deprecated and is not used wouldn't hurt anything to prevent this from being a breaking change.

Reply:
Went with that approach, is nicer for users. Thanks!